helmfiles/helmfile.yaml

```diff
@@ -2,11 +2,25 @@
 releases:
     - name: bitcoind
       chart: ../charts/bitcoind
+      values:
+          - bitcoindHostPath: {{env "PWD"}}/volumes/bitcoind
     - name: lnd1
       chart: ../charts/lnd
+      values:
+          - gRPCNodePort: 30009
+          - lndHostPath: {{env "PWD"}}/volumes/lnd1
     - name: lnd2
       chart: ../charts/lnd
+      values:
+          - gRPCNodePort: 30010
+          - lndHostPath: {{env "PWD"}}/volumes/lnd2
     - name: lnd3
       chart: ../charts/lnd
+      values:
+          - gRPCNodePort: 30011
+          - lndHostPath: {{env "PWD"}}/volumes/lnd3
     - name: lnd4
       chart: ../charts/lnd
+      values:
+          - gRPCNodePort: 30012
+          - lndHostPath: {{env "PWD"}}/volumes/lnd4
```

Comment:
This should probably be changed. I was not able to make this path relative to the helmfile or the place the user rand the command from. So I ended up with this solution. The problem here is that PWD does not exist on Windows (afaik.).